### PR TITLE
CVSB-12674

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

**Comment by vladddragomir** [comment]
some of them are, some of them are not, but in case the migrated data comes without being aligned with the api specs this should prevent the app from crashing

File: src/pages/testing/vehicle/vehicle-additional/vehicle-additional.html [hunk 1]
```diff
@@ -1,322 +1,329 @@
...
+      <div item-content padding-right>
+        <p>{{ vehicleData.techRecord.rearAxleToRearTrl }}</p>
+      </div>
+    </ion-item>
+
+  </ion-list>
+
+  <ion-list class="vehicle-list" *ngIf="commonFunc.checkForMatch(vehicleData.techRecord.vehicleType,VEHICLE_TYPE.TRL)">
+    <ion-list-header padding-left padding-right text-wrap class='list-header-double'>Coupling centre to the rear axle
+      centre (mm)
```
Comment (KHodderDVSA) [inline src/pages/testing/vehicle/vehicle-additional/vehicle-additional.html]: If you're breaking for line length, perhaps better to put the entire body string on a new line?

File: src/pages/testing/vehicle/vehicle-additional/vehicle-additional.html [hunk 2]
```diff
@@ -1,322 +1,329 @@
...
+      <div item-content padding-right>
+        <p>{{ vehicleData.techRecord.couplingCenterToRearAxleMax }}</p>
+      </div>
+    </ion-item>
+
+  </ion-list>
+
+  <ion-list class="vehicle-list" *ngIf="commonFunc.checkForMatch(vehicleData.techRecord.vehicleType,VEHICLE_TYPE.TRL)">
+    <ion-list-header padding-left padding-right text-wrap class='list-header-double'>Coupling centre to the rear of the
+      trailer (mm)
```
Comment (KHodderDVSA) [inline src/pages/testing/vehicle/vehicle-additional/vehicle-additional.html]: same here

File: src/pages/testing/vehicle/vehicle-additional/vehicle-additional.html (hunk 1 above)
Comment (vladddragomir) [reply src/pages/testing/vehicle/vehicle-additional/vehicle-additional.html]: it's because of the IDEA's auto-indentation

**Pull request merged**